raster-foundry/blasterjs
# Rename Header to Heading

## Overview

Renames `Header` to `Heading` to better align with HTML lingo.

### Checklist



Are there any of the following in this PR?




If one of the above is checked...

- [ ] information or guidance around needed changes for consumers of this library has been added to the changelog under `Upgrade Instructions`


## Testing Instructions

 * Run docz. Confirm Typography/Heading works.
 * Search codebase. Confirm all instances of "header" and "Header" are unrelated to the Heading (nee Header) component.
